src/fides/api/alembic/migrations/versions/d0cbfec0b2dd_add_okta_to_connection_types.py

```diff
@@ -0,0 +1,61 @@
+"""add okta to connection_types
+
+Revision ID: d0cbfec0b2dd
+Revises: c9c72b3d550b
+Create Date: 2025-04-23 15:22:57.920715
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "d0cbfec0b2dd"
+down_revision = "c9c72b3d550b"
+branch_labels = None
+depends_on = None
+
+type_to_handle = "okta"
+
+
+def get_enum_values():
+    conn = op.get_bind()
+    result = conn.execute("SELECT unnest(enum_range(NULL::connectiontype))").fetchall()
+    return [row[0] for row in result]
+
+
+def upgrade():
+    # add 'okta' to ConnectionType enum
+    enum_values = [f"'{v.strip()}'" for v in get_enum_values() + [type_to_handle]]
```

Comment:
No reason to have this as an fstring right? We could just do `str(v.strip())` or `str(v)` given the suggestion in `get_enum_values`. Small beans but I think we should keep this line as clean as possible